### DISCUSSION TITLE
help with development installation of pydeck

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#6076] [Q&A]
Hi, 

I'm trying to follow the instructions in the [pydeck docs](https://deckgl.readthedocs.io/en/latest/contributing.html) about how to set up a development installation of pydeck. However, I'm getting an error at the 

```
make prepare-jupyter 
```

step. that says:

```
ERROR: Package 'pydeck' requires a different Python: 3.6.5 not in '>=3.7'
```

I had set up and activated a new conda environment with Python 3.6.5 using 

```
conda create -n py365 python=3.6.5
conda activate py365
```

such that running Python in the environment gives

```
python
Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

I'm using MacOS 11.5.1

**Comment by kylebarron** [comment]
> ```
> ERROR: Package 'pydeck' requires a different Python: 3.6.5 not in '>=3.7'
> ```

This error should tell you what's wrong... Pydeck requires Python 3.7 and you're on 3.6
https://github.com/visgl/deck.gl/blob/62c579bab4114b34e7ac7fbeb8e05fc0ea11c13b/bindings/pydeck/setup.py#L196

**Reply by cornhundred** [reply]
Thanks @kylebarron, but I'm seeing the same error if I set up a Python 3.7 conda environment

```
ERROR: Package 'pydeck' requires a different Python: 3.6.5 not in '>=3.7'
make: *** [init] Error 1
```

I setup 3.7 like this

```
conda create -n py37 python=3.7
conda activate py37
```

and running Python shows this

```
$python
Python 3.7.11 (default, Jul 27 2021, 07:03:16) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

**Reply by kylebarron** [reply]
That implies that the Python/pip found on your PATH still isn't finding the one in the 3.7 environment.
